ee/clickhouse/queries/funnels/funnel_time_to_convert.py

```diff
@@ -60,10 +60,19 @@ def get_query(self) -> str:
         ]
         steps_average_conversion_time_expression_sum = " + ".join(steps_average_conversion_time_identifiers)
 
+        steps_average_conditional_for_invalid_values = [
+            f"{identifier} >= 0" for identifier in steps_average_conversion_time_identifiers
```

Comment:
Are NULLs an issue, should we add a NOT NULL check before >= 0?

Reply:
NULL values ought to be removed, and this check implies NULL values won't make it through, either!

This is kinda tested by existing tests: `test_auto_bin_count_total` - step_1 time is > 0, while step_2 is NULL, and as expected, it's removed from consideration.